lib/utils/compose_ts.ts

```diff
@@ -14,13 +14,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import * as _ from 'lodash';
 import * as Bluebird from 'bluebird';
```

Comment:
I know this probably isn't your doing, but I really don't like some source files referencing Bluebird as `Promise` and others as `Bluebird`. It's confusing at first glance and anyone seeing it imported as `Bluebird` here might assume other files with `Promise` in are native.